examples/api/README.md

```diff
@@ -77,8 +77,13 @@ Use `{@tool dartpad}` for Dartpad examples, and use `{@tool sample}` for
 examples that shouldn't be run/shown in Dartpad.
 
 Once that comment block is inserted in the source code, create a new file at the
-appropriate path under [`examples/api`](.). You should also add tests for your
-sample code under [`examples/api/test`](./test).
+appropriate path under [`examples/api`](.) that matches the location of the
+source file they are linked from, and are named for the symbol they are attached
+to, in lower_snake_case, with an index relating to their order within the doc
+comment. So, for the `Curve2D` case, since it's in the `animation` package, in a
+file called `curves.dart`, and it's the first example, it goes in
+`examples/api/lib/animation/curves/curve2_d.0.dart`. You should also add tests 
```

Comment:
Looks good.  Can you put the "You should also add tests" line in a separate paragraph?